### ISSUE TITLE
main.js not found

### ISSUE THREAD

**Issue by jazzdan** [knowbody/crra#5]
Hi!

Thanks for putting this together! After running `yarn` to install the dependencies, I get a suspicious error message when I run `yarn dev`:

```
ERROR in multi (webpack)-dev-server/client?http://localhost:8080 ./lib/js/src/index
Module not found: Error: Can't resolve './lib/js/src/index' in '/Users/dan/development/reason/crra'
 @ multi (webpack)-dev-server/client?http://localhost:8080 ./lib/js/src/index
```

However, it appears to finish. But when I visit http://localhost:8080 in my browser I see that http://localhost:8080/main.js  returns a 404.

I'm running node v8.0. Happy to provide any other info, just let me know!

Thanks again!

**Comment by msenevir** [comment]
I noticed this as well, thanks for opening an issue. Pushing up a fix for this shortly

**Comment by jazzdan** [comment]
Awesome, thank you! :D

**Comment by knowbody** [comment]
fixed by #6

thank you @msenevir !
